insights/tools/insights_inspect.py

```diff
@@ -25,9 +25,9 @@
 
 >>> insights-inspect insights.combiners.hostname.hostname
 IPython Console Usage Info:
-Enter 'make_pass.' and tab to get a list of properties
+Enter 'Hostname.' and tab to get a list of properties
```

Comment:
Thanks for catching this @iNecas  
This should actually be ...
Enter 'hostname.'

Reply:
```
insights-inspect insights.parsers.hostname.Hostname ~/tmp/ccx/sosreport-luna122-1-2019-08-19-uakvnas.tar.xz

IPython Console Usage Info:

Enter 'Hostname.' and tab to get a list of properties 
Example:
In [1]: Hostname.<property_name>
Out[1]: <property value>

To exit ipython enter 'exit' and hit enter or use 'CTL D'

Starting IPython Interpreter Now 

```

and I actually can use `Hostname`, while `hostname` would not work.

Reply:
Scratch that, I've missed there is different input here. Will update

Reply:
Nice catch, updated